### DISCUSSION TITLE
RPM uses pragma case_sensitive_like which is deprecated from sqlite 3.44

### DISCUSSION THREAD

**Discussion by vbrahmajosyula1** [rpm-software-management/rpm#2924] [General]
sqlite 3 deprecates pragma_case_sensitive_like from 3.44 https://www.sqlite.org/changes.html.
Following patch introduced case_sensitive_like in RPM.
https://github.com/rpm-software-management/rpm/commit/bea298986d796900dd94de42dec98fb194df1ff8

Are there active plans for a replacement?

**Comment by pmatilai** [comment]
No, because I had no idea. I don't particularly follow sqlite developments because there aren't any active developments or plans in that direction. But clearly we need to do something about it sooner or later then, filed https://github.com/rpm-software-management/rpm/issues/2925 so it gets tracked.

Thanks for the heads-up! :+1:
